website/blog/2024-08-08-release-1.12.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Podman Desktop 1.12 Release
+description: Podman Desktop 1.12 has been released!
+slug: podman-desktop-release-1.12
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.12/banner.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.12 Release! 🎉
+
+![podman-desktop-hero-1.12](img/podman-desktop-release-1.12/banner.png)
+
+Podman Desktop 1.12 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **Podman remote**: We now support remote Podman setups! Manage your remote Podman machines all within the UI.
+- **macOS GPU support**: Container GPU access on macOS is now available. [`libkrun`](https://github.com/containers/libkrun) is now a selectable provider type to allow GPU passthrough enablement.
+- **Podman 5.2.0**: This new version of Podman provides GPU access for macOS, as well as a host of [new features](https://github.com/containers/podman/releases/tag/v5.2.0).
+- **Light mode out of experimental**: Our light mode has been well-received, and we have now marked it as non-experimental! Enjoy the new theme.
```

Comment:
Other things to callout:
- more kubernetes objects supported too: ConfigMaps, Secrets.
- Improved font consistency
- Display Image Files

Reply:
Done!